proof/invariant-abstract/X64/ArchBits_AI.thy

```diff
@@ -10,6 +10,14 @@ begin
 
 context Arch begin arch_global_naming
 
+(* arch-specific interpretations of update locales: *)
+
+(* FIXME: do this for other x64 arch_state fields *)
```

Comment:
is this an arch-split, or do you think it can be left for arch-local improvements if they ever happen for x64?

Reply:
I think the latter. See https://github.com/seL4/l4v/blob/78ad44e4a5594df14a4e50fe17680df2f1eeb4e0/proof/invariant-abstract/AARCH64/ArchBits_AI.thy#L14 for what I was copying from AARCH64. It would probably be fine to do the same for all of the arch_state fields on X64 but there's quite a lot of them and at this stage I just wanted to get my current proofs through, so I just did the one I need now.